app/src/main/java/com/xmartlabs/scasas/geoquiz/QuizActivity.java

```diff
@@ -0,0 +1,53 @@
+package com.xmartlabs.scasas.geoquiz;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+public class QuizActivity extends AppCompatActivity {
+    private Button mTrueButon;
```

Comment:
The variables should not have the prefix, review [this card](https://trello.com/c/Ismail5p)